recipe/meta.yaml

```
@@ -6,68 +6,92 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nannyml-{{ version }}.tar.gz
-  sha256: db0e4a688d9d830ad3c461106f42335588ee03241f16a00fde00767b408ef704
+  url: https://github.com/NannyML/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 67801824c003ec37b055a6c0f4d540e20d3eff8f84cb6e202a40d2b349c092df
 
 build:
+  number: 0
   entry_points:
     - nannyml=nannyml.cli.cli:cli
     - nml=nannyml.cli.cli:cli
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # https://github.com/NannyML/nannyml/blob/v0.12.1/pyproject.toml#L33
+  skip: True  # [py<38 or py>311]
+  # nothing provides requested flaml >=1.0.11,<2.0 on s390x
+  # nothing provides requested python-kaleido 0.2.1.* on s390x
+  skip: True  # [s390x]
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.9,<3.12
+    - python
   run:
-    - python >=3.9,<3.12
-    - pandas >=1.5.0,<2.0.0
+    - python
+    - pandas >=1.5,<2
     - python-dateutil >=2.8.2,<3.0.0
-    - scikit-learn >=1.2,<2.0.0
-    - joblib >=1.1.0,<2.0.0
+    - scikit-learn >=1.2,<2.0
+    - joblib >=1.1.0,<2.0
     - category_encoders >=2.3.0,<3.0.0
-    - matplotlib-base >=3.7.0,<4.0.0
+    - matplotlib >=3.7,<4.0
     - plotly >=5.6.0,<6.0
-    - scipy >=1.10.0,<2.0.0
+    - scipy >=1.10,<2.0
     - numpy >=1.24,<1.25
-    - seaborn >=0.13.2,<0.14.0
+    - seaborn >=0.13.2,<1.0
+    # kaleido same as python-kaleido
     - python-kaleido ==0.2.1
-    - pyarrow <15.0.0,>=14.0.0
-    - gcsfs >=2022.5.0,<2023.0.0
+    - pyarrow >=14.0.0,<15.0.0
+    - gcsfs >=2022.5.0
     - pydantic >=2.7.4,<3.0.0
-    - rich >=12.5.1,<13.0.0
+    - rich >=12.5.1
     - click >=8.1.3,<9.0.0
     - pyyaml >=6.0,<7.0
     - jinja2 <3.1
-    - pyfiglet >=0.8.post1,<0.9
-    - lightgbm >=3.3.2,<4.0.0
-    - flaml >=1.0.11,<2.0.0
-    - s3fs >=2022.8.2,<2023.0.0
-    - apscheduler >=3.9.1,<4.0.0
-    - analytics-python >=2.3.2,<3.0.0
-    - python-dotenv >=0.21.0,<0.22.0
+    - pyfiglet >=0.8.post1,<1.0
+    - lightgbm >=3.3.2,<4.0
+    - flaml >=1.0.11,<2.0
+    - s3fs >=2022.8.2
+    - apscheduler >=3.9.1,<4.0
+    - analytics-python >=2.3.2,<3.0
+    - python-dotenv >=0.21.0,<1.0
     - types-pyyaml >=6.0.12.8,<7.0
     - types-python-dateutil >=2.8.19.10,<3.0
+  run_constrainted:
+    - sqlmodel
+    # psycopg2 on defaults is basically psycopg2-binary (without the bundled/vendored dependencies)
+    - psycopg2
```

```suggestion
    - psycopg2 >=2.9.3,<3
```
https://github.com/NannyML/nannyml/blob/v0.12.1/pyproject.toml#L56C24-L58